app/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -203,23 +172,21 @@ export default class DateTimePicker extends React.Component {
       <div className={containerStyle.getClasses()}>
         <div className={styles.head}>
           <button
-            className={styles.arrow}
+            className={`${styles['page-icon']} ${styles['page-icon-prev']}`}
             onClick={this.handleMonthChange.bind(this, -1)}
```

Comment:
Whenever I work on a file and I run into a function that is created and then garbage-collected with every re-render, I'd tidy that up into the constructor, i.e. something like
```
this.goToPrevMonth = this.handleMonthChange.bind(this, -1)
this.goToNextMonth = this.handleMonthChange.bind(this, 1)
```

Reply:
Done